### PR TITLE
Improve README

### PR DESCRIPTION
**Pull request by wo0dyn**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -54,31 +54,32 @@ Lets say you're using a lib named ``umbrella`` which provides a function named
...
         umbrella.put_pouch()
 
-This function does what it says it does, but it's not ideally splitted, depending on
+
+This function does what it says it does, but it's not ideally split, depending on
 your needs. For example, maybe at some point you realize you need each of the 3 separate
-parts to be a function of its own. Or maybe you can't call time.sleep in your app. Or do
-something else with the ``umbrella`` when it's open like dance with it.
+parts to be a function of its own. Or maybe you can't call ``time.sleep`` in your app. Or do
+something else with the ``umbrella`` when it's opened like dance with it.
```
Comment (ojh) [inline README.rst:61]:
```suggestion
something else with the ``umbrella`` when it's opened, like dance with it.
```

File: README.rst [hunk 2]
```diff
@@ -54,31 +54,32 @@ Lets say you're using a lib named ``umbrella`` which provides a function named
...
 
 It's also possible that you can't really make a pull request because your needs are
 specific, or you don't have the time (that's sad but, hey, I know it happens) or any
-other personnal reason. So what do you do? There's no real alternative. You copy and
-paste the code, modify it to fit your needs and use your modified version. And whenever
+other personal reason. So what do you do? There are no real alternative. You copy/paste
```
Comment (ojh) [inline README.rst:65]:
```suggestion
other personal reason. So what do you do? There's no real alternative. You copy/paste
```

File: README.rst [hunk 3]
```diff
@@ -54,31 +54,32 @@ Lets say you're using a lib named ``umbrella`` which provides a function named
...
+the code, modify it to fit your needs and use your modified version. And whenever
 there's a change to the upstream function, chances are you'll never know.
 
 
 The solution
 ------------
 
-Enter Raincoat.
+*Enters Raincoat.*
```
Comment (ojh) [inline README.rst:73]:
```suggestion
*Enter Raincoat.*
```

File: README.rst [hunk 4]
```diff
@@ -114,22 +115,22 @@ Now, install and run ``raincoat`` in your project:
...
+#. Look at the currently installed version of the lib (say, umbrella 16.0.3) (or, if not
+   found, the latest version)
+#. Compare with the version in the Raincoat comment (here, 14.5.7)
+#. If they are different, download and pip install the specified version in a temp dir
+   (using cached wheel as pip does by default, this should be quite fast in most cases)
+#. Locate the code using the provided path for both the downloaded and the currently
+   installed versions
+#. Diff it
+#. Tell you if there's a difference (and mention the location of the original Raincoat
+   comment)
```
Comment (ojh) [inline README.rst:133]: I would keep the colon and then indent the following items.
```
- Grep the code for all ``# Raincoat:`` comments and for each comment:
  #. Look at the currently installed version of the lib (say, umbrella 16.0.3) (or, if not
     found, the latest version)
  #. Compare with the version in the Raincoat comment (here, 14.5.7)
  #. If they are different, download and pip install the specified version in a temp dir
     (using cached wheel as pip does by default, this should be quite fast in most cases)
  #. Locate the code using the provided path for both the downloaded and the currently
     installed versions
  #. Diff it
  #. Tell you if there's a difference (and mention the location of the original Raincoat
     comment) ```

File: README.rst [hunk 5]
```diff
@@ -25,16 +25,16 @@ Raincoat
     :target: CODE_OF_CONDUCT.md
     :alt: Contributor Covenant
 
-Raincoat has you covered when you can't stay DRY. When the time comes where you HAVE to
+Raincoat keeps you covered when you can't stay DRY. When the time comes where you HAVE to
```
[the remark this answers is not in the document]
Comment (wo0dyn) [reply README.rst:28]: Ah my bad. Thanks. 😊

File: README.rst (hunk 4 above)
Comment (wo0dyn) [reply README.rst:133]: Agree. 👍

File: README.rst (hunk 5 above)
[the remark this answers is not in the document]
Comment (ewjoachim) [reply README.rst]: Yeah the idea here really is "Raincoat has your back", but the joke with a raincoat covering you. I'm not sure it works as well with "keeps".

**Comment by ewjoachim** [comment]
The doc is currently very very very lacking, if you want to tackle that :)

**Pull request merged**